Zilon.Core/Zilon.Bot.Players.DevelopmentTests/BotActorTaskSourceTests.cs

```diff
@@ -32,6 +31,10 @@ public class BotActorTaskSourceTests
         [TestCase("monster")]
         public async Task GetActorTasksTestAsync(string mode)
         {
+            var _globalServiceContainer = new ServiceCollection();
+            var startUp = new Startup();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'startUp'.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5014856)